src/pages/Powers/Index.js

```diff
@@ -7,10 +7,6 @@ import ComponentTabs from '@/components/ComponentTabs';
 import DerivativeComponent from '@/components/DerivativeComponent'
 import ExchangeTVChart from '@/components/ExchangeTVChart/ExchangeTVChart';
 import { PositionTable } from '@/components/OptionComponent/TableComponent';
-import * as Api from '@/acy-dex-futures/Api';
-import useSWR from 'swr'
```

Comment:
This merge commit seems to break the Powers page. In this file Line 86 it still uses `useSWR`.